stubs/pyflakes/pyflakes/checker.pyi

```diff
@@ -154,42 +152,42 @@ if sys.version_info >= (3, 10):
     _MatchAs: TypeAlias = ast.MatchAs
     _MatchOr: TypeAlias = ast.MatchOr
 else:
-    _Match: TypeAlias = Any
-    _MatchCase: TypeAlias = Any
-    _MatchValue: TypeAlias = Any
-    _MatchSingleton: TypeAlias = Any
-    _MatchSequence: TypeAlias = Any
-    _MatchStar: TypeAlias = Any
-    _MatchMapping: TypeAlias = Any
-    _MatchClass: TypeAlias = Any
-    _MatchAs: TypeAlias = Any
-    _MatchOr: TypeAlias = Any
+    _Match: TypeAlias = Incomplete
+    _MatchCase: TypeAlias = Incomplete
+    _MatchValue: TypeAlias = Incomplete
+    _MatchSingleton: TypeAlias = Incomplete
+    _MatchSequence: TypeAlias = Incomplete
+    _MatchStar: TypeAlias = Incomplete
+    _MatchMapping: TypeAlias = Incomplete
+    _MatchClass: TypeAlias = Incomplete
+    _MatchAs: TypeAlias = Incomplete
+    _MatchOr: TypeAlias = Incomplete
```

Comment:
I think these can all remain as `Any` with a comment (and the same for the `if sys.version_info >= (3, 12):` branch below, too). The methods these type aliases are used to annotate (`visit_MatchAs`, etc.) would never be called on Python <=3.9, and if you did explicitly call them on Python <=3.9 for some reason, it would be impossible to pass them an argument that would result in sane behaviour. I suppose `Never` could also be fine as the value for the aliases, though it doesn't seem too important to change it from `Any`